scripts/deploy/config.ts

```diff
@@ -19,29 +19,22 @@ type TokenSaleConfig = {
   TOKEN_SALE_ARENA_PRICE: BN;
   TOKEN_SALE_RECIPIENT: string;
   TOKEN_SALE_WHITELIST: typeof TOKEN_SALE_WHITELIST;
+  RECIPIENT_AMOUNT: BN;
 };
 
+// TODO: update values as they are in USDC when they should be in ARENA tokens
```

Comment:
this is an important `TODO`, especially as USDC uses 6 decimals, so the values below are currently wrong.